docs/troubleshoot/troubleshoot-apiml.md

```diff
@@ -547,3 +547,57 @@ If you are using Zowe's local CA certificate and you still receive **ZWED0148E**
 ```
 
 In this case, ensure that the label names exactly match the names in TSO when confirming your keyring. Any difference in spaces, capitalization, or other places throw the error.
+
+### ZWEAM400E Error initializing SSL Context when using Java 11
+
+**Symptom:**
+
+API ML components configured with SAF keyring are not able to start due to an unrecoverable exception. The message indicates that `safkeyring` is unknown protocol. 
+
+**Examples:**
+```
+2023-06-27 13:07:04.493 <ZWEAGW1:main:394418> ZWESVUSR ERROR (o.z.a.s.HttpsFactory) error java.net.MalformedURLException: unknown protocol: safkeyring                                           
```

Comment:
Where is this example/error message thrown?

Reply:
All these messages appear in the Zowe STC joblog.

Reply:
Ok.